packages/fxa-content-server/app/scripts/lib/router.js

```diff
@@ -269,16 +286,35 @@ Router = Router.extend({
       });
     },
     'oauth(/)': createViewHandler(IndexView),
-    'oauth/force_auth(/)': createViewHandler(ForceAuthView),
-    'oauth/signin(/)': function () {
-      this.createReactOrBackboneViewHandler('signin', SignInPasswordView, {
-        // see comment in fxa-settings/src/pages/Signin/container.tsx for param explanation
+    'oauth/force_auth(/)': function () {
+      this.createReactOrBackboneViewHandler('oauth/force_auth', ForceAuthView, {
         ...Url.searchParams(this.window.location.search),
         email: this.user.get('emailFromIndex'),
         hasLinkedAccount: this.user.get('hasLinkedAccount'),
         hasPassword: this.user.get('hasPassword'),
+        // for subplat redirect only
+        ...(this.relier.get('redirectTo') && {
+          redirect_to: this.relier.get('redirectTo'),
+        }),
       });
     },
+    'oauth/signin(/)': function () {
+      this.createReactOrBackboneViewHandler(
+        'oauth/signin',
+        SignInPasswordView,
+        {
+          // see comment in fxa-settings/src/pages/Signin/container.tsx for param explanation
+          ...Url.searchParams(this.window.location.search),
```

Comment:
Is there an instance where we wouldn't the React view to get all the `this.window.location.search` params?